docs/source/app_developers_guide/docker.rst

```diff
@@ -141,11 +142,24 @@ To start the Sawtooth Docker environment, perform the following tasks:
 
    .. code-block:: console
 
-     user@host$ docker-compose -f sawtooth-default.yaml up
+     user@host$ sudo docker-compose -f sawtooth-default.yaml up
```

Comment:
We’ve been trying to avoid requiring the user making any calls with `sudo`. As far as I know, none of the team needs to do this, and your justification for doing this is preventing an error message that has to do with a users local docker setup, which it is not this document’s responsibility to train users on how to use docker.

Reply:
That effectively requires giving root access to the client user, without the auditing benefits of sudo. The error message was there for a reason and there's a reason most or all distributions require using sudo with docker.

But I will make the change you suggest and add a note assuming an insecure setup.